app/models/guide.rb

```diff
@@ -84,6 +84,8 @@ def slug_format
       errors.add(:slug, "must be filled in")
     elsif !slug.to_s.match(/\A\/service-manual\/[a-z0-9\-\/]+$/i)
       errors.add(:slug, "can only contain letters, numbers and dashes")
+    elsif !slug.to_s.match(/\A\/service-manual\/[a-z0-9-]+\/[a-z0-9-]+/)
+      errors.add(:slug, "must be present and start with '/service-manual/[TOPIC]'")
```

Comment:
I reckon this is the right approach for the moment.

We could check for the presence of the actual topic name _or even_ generate the whole prefix and only let the user edit the last part of the slug. The user can change the slug right up until publishing. Too much at the moment though I reckon.

You could use curly brackets for repetition in that regex. eg. `/\A\/service-manual(\/[a-z0-9-]+){1,2}`